packages/rule-no-p3p/tests/no-p3p.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint sort-keys: 0, no-undefined: 0 */
+
+import { generateHTMLPage } from 'sonarwhal/dist/tests/helpers/misc';
+import { RuleTest } from 'sonarwhal/dist/tests/helpers/rule-test-type';
+import * as ruleRunner from 'sonarwhal/dist/tests/helpers/rule-runner';
+
+const ruleName = 'no-p3p';
```

Comment:
`import { getRuleName } from 'sonarwhal/dist/src/lib/utils/rule-helpers';`
and
`const ruleName = getRuleName(__dirname);`